LSLib/Granny/GR2/Format.cs

```diff
@@ -524,7 +524,7 @@ public enum SectionType : uint
         TrackGroup = 1,
         Skeleton = 2,
         Mesh = 3,
-        CurveAndDiscardable = 4,
+        StructDefinitions = 4,
```

Comment:
This change isn't strictly necessary but I thought it might make things slightly less confusing since it's no longer where curve data is being serialized. I'm also not sure if this is the best name so it might make more sense to just leave it as-is.